src/index.js

```diff
@@ -12,8 +12,8 @@ canvas.height = CANVAS_HEIGHT;
 
 // Generate a new world
 const world = new World();
-world.generate();
 
 // Create a new renderer and start the animation
 const render = new Render(canvas, world);
 render.animate();
+world.generate();
```

Comment:
Why did you move this line here?
It makes more sense to me to first add objects to the world, then start animating it.

Reply:
No idea how it got swapped, corrected.